ci-operator/step-registry/agent/ibmz/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+- v78singh
```

Comment:
I'm fine with landing this as is, but can we have a follow up patch to add:
- multiarch-approvers
and
- multiarch-reviewers

to all of the added sections?

Reply:
Sure @jaypoulz, Please let me know whom to add

Reply:
You would add literally what he typed above.  In the `approvers:` section, add `- multiarch-approvers`, and the same for the reviewers section.

Reply:
Added the multiarch-approvers and reviewers in the OWNERS, please re-review :)

Reply:
yeah @tvardema, I understood that later on when I was checking other OWNERS files, Earlier I thought some names will be pointed out 😅